apps/igloo-kit-cli/src/cli/mod.rs

```diff
@@ -81,20 +81,33 @@ impl CommandTerminal {
 }
 
 async fn top_command_handler(term: &mut CommandTerminal, config: Config, commands: &Option<Commands>, debug: bool) {
-
-    if (!config.features.coming_soon_wall) {
+    let clickhouse_config = ClickhouseConfig {
```

Comment:
Feels like we will need to eventually make those parameters?

Reply:
nah, bad form in rust and the linter was complaining